RileyLinkKitTests/PumpOpsSynchronousBuildFromFramesTests.swift

```diff
@@ -74,16 +74,14 @@ class PumpOpsSynchronousBuildFromFramesTests: XCTestCase {
         assertThrows(try _ = sut.getHistoryEvents(since: Date()))
     }
     
-    func testAllEventsReturned() {
+    func test332EventsReturnedUntilOutOrder() {
```

Comment:
Is this testing a discontinuity?  Do you have one of those as a pump history page capture?